perllib/FixMyStreet/Cobrand/Bristol.pm

```diff
@@ -162,4 +168,101 @@ sub post_report_sent {
     }
 }
 
+=head2 responsible_for_areas
+
+The Bristol body is configured to cover some other areas, but is only responsible for problems in parks in those areas.
+
+Override this method so an error is shown if the user tries to report a problem outside Bristol.
+
+=cut
+
+sub responsible_for_areas {
+    my ($self, $areas) = @_;
+
+    if (scalar(%$areas) == 1 && (values %$areas)[0]->{name} eq $self->council_name) {
```

Comment:
Is there every a possibility for $areas to not only be 1 in length here? I think it will only be one long.
Could go on the ID matching rather than the name, though unlikely to change I guess.